hammer_time/tests/test_hammer_time.py

```diff
@@ -174,6 +175,31 @@ def perform(cls, client):
     return cls.perform(client, **parameters)
 
 
+class TestRunAvailable(TestCase):
+
+    def test_iter_blocking_state(self):
+        client = fake_juju_client()
+        ra = RunAvailable(client, '0')
+        with patch.object(client._backend, 'juju',
+                          return_value=1) as juju_mock:
```

Comment:
I would have used
    side_effect=(1, 0)
but in your assertions I think your stepped approach makes it clear that list(ra.iter_blocking_state(None)) returns a list of blocked machines or an empty list.